skyportal/handlers/api/observation_plan.py

```diff
@@ -71,72 +71,77 @@ def post(self):
                               description: New observation plan request ID
         """
         data = self.get_json()
```

Comment:
maybe change the first instance of `data` to `json`.

Reply:
good idea.